### ISSUE TITLE
[Bug]: focus indicator is not visible in iframes/child windows

### ISSUE THREAD

**Issue by layershifter** [microsoft/fluentui#27197]
### Library

React Components / v9 (@fluentui/react-components)

### System Info

```shell
N/A
```

### Are you reporting Accessibility issue?

no

### Reproduction

https://codesandbox.io/s/suspicious-water-0rv2et

## Actual Behavior

- Open a CodeSandbox
- Click on the first button inside an iframe
- Press <kbd>Tab</kbd>
- Focus indicator is invisible 💥

## Expected Behavior

- Focus indicator should be visible

### Logs

```shell
N/A
```

### Requested priority

Blocking

### Products/sites affected

Teams

### Are you willing to submit a PR to fix?

yes

### Validations

- [X] Check that there isn't already an issue that reports the same bug to avoid creating a duplicate.
- [X] The provided reproduction is a minimal reproducible example of the bug.
